config/routes.rb

```diff
@@ -1,8 +1,28 @@
 Rails.application.routes.draw do
   get 'placements_imports/new'
   get 'placements_imports/create'
-  devise_for :users
+  
+  delete 'admins/:id' => 'admins#destroy', :as => :admin_destroy_user
```

Comment:
You can write this in a slightly more Rails-y way:

```suggestion
  delete 'admins/:id', to: 'admins#destroy', as: :admin_destroy_user
```

Reply:
Done.